parm/wflow/plot.yaml

```
@@ -12,37 +12,45 @@ default_task_plot: &default_task
     PDY: !cycstr "@Y@m@d"
     cyc: !cycstr "@H"
     subcyc: !cycstr "@M"
+    fhr: '#fhr#'
     LOGDIR: !cycstr "&LOGDIR;"
     SLASH_ENSMEM_SUBDIR: '&SLASH_ENSMEM_SUBDIR;'
     ENSMEM_INDX: '#mem#'
-    nprocs: '{{ nnodes * ppn }}'
+    nprocs: '{{ parent.nnodes * parent.ppn }}'
+  join: !cycstr '&LOGDIR;/{{ jobname }}_@Y@m@d@H&LOGEXT;'
   native: '{{ platform.SCHED_NATIVE_CMD }}'
-  nnodes: 1
+  nnodes: 2
```

Is the plotting script MPI capable? If not, this is wasting resources that will never be used.

---

Please change the 2 here back to 1.

```suggestion
  nnodes: 1
```